identity-apps-core/apps/authentication-portal/src/main/webapp/includes/footer.jsp

```diff
@@ -17,3 +17,118 @@
   --%>
 
 <script src="libs/themes/default/semantic.min.js"></script>
+
+<%-- App Insights --%>
+<%
+    if (config.getServletContext().getResource("extensions/app-insights.jsp") != null) {
+%>
+        <jsp:include page="/extensions/app-insights.jsp"/>
```

Comment:
App insights need to be loaded through extensions/analytics.jsp file. The extension point is already defined in the other pages, therefore we don't need this here.